styles/signals.json

```diff
@@ -99,6 +99,19 @@
 			}],
 			"caption": "Transmission voie-machine (TVM)"
 		},
+        {
```

Comment:
The indentation looks wrong compared to the rest of the file. I suspect your editor uses spaces instead of tabs and a tab width of 4.

Reply:
You're right - fixed it!